stream/microsoft-defender-intel/src/config.yml.sample

```
@@ -3,12 +3,18 @@ opencti:
   token: 'ChangeMe'
 
 connector:
-  id: 'ChangeMe'
-  type: 'STREAM'
-  name: 'Microsoft Defender Intel'
-  live_stream_id: 'ChangeMe'
+  #id: '4aa40d9f-bee0-466e-a72e-6878c13bde08' # Optional (default: '4aa40d9f-bee0-466e-a72e-6878c13bde08')
+  #name: 'Microsoft Defender Intel' # Optional (default: 'Microsoft Defender Intel')
+  #scope: 'defender' # Optional (default: 'defender')
+  #live_stream_id: 'live' # Optional (default: 'live')
```

A stream ID must be explicitly set with appropriate filters.
Using "live" as the default is risky, as it would collect all events from every action on the OpenCTI platform.

```suggestion
  #live_stream_id: 'ChangeMe'
```